src/Pact/Compile.hs

```diff
@@ -513,3 +606,6 @@ _atto fp = do
 
 _testCToTV :: Bool
 _testCToTV = nub vs == vs where vs = take (26*26*26) $ map cToTV [0..]
+
+_unusedLenses :: Compile ()
+_unusedLenses = void $ return (set msName,set msHash)
```

Comment:
But why are they unused?

Reply:
cuz they're unused in the module and not exported. alternative is to not use `makeLenses`